tracer/src/Datadog.Trace/Debugger/Instrumentation/AsyncLineDebuggerInvoker.cs

```diff
@@ -147,7 +146,9 @@ private static void CollectArgs(ref AsyncLineDebuggerState asyncState)
             for (var index = 0; index < kickOffMethodArguments.Length; index++)
             {
                 ref var argument = ref kickOffMethodArguments[index];
-                if (argument == default)
+                if (argument == default ||
```

Comment:
This predicate is repeated twice, maybe extract method just in case?

Reply:
Fixed in d93f31f39ab9606ae745e1c8547fc5e7e0a6b757